cloud/gcp/modules/storage/outputs.tf

```diff
@@ -1,7 +1,7 @@
-output "storage_name" {
-  value = google_storage_bucket.object_storage.name
+output "bucket_name" {
+  value = google_storage_bucket.file_storage.name
 }
 
 output "storage_id" {
-  value = google_storage_bucket.object_storage.id
+  value = google_storage_bucket.file_storage.id
 }
```

Comment:
nit: here and other files - newline at the end.

Reply:
Fixed. terraform fmt somehow removes the newlines at the end.